commit 87d9e3c4adac92ff544440dd1239a4ae4fe05bb4
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Mon Nov 28 14:00:45 2011 +0200

    Fix classification of ELF binaries with setuid/setgid bit, oops...

diff --git a/fileattrs/elf.attr b/fileattrs/elf.attr
index bc6ce83..595b33e 100644
--- a/fileattrs/elf.attr
+++ b/fileattrs/elf.attr
@@ -1,4 +1,4 @@
 %__elf_provides		%{_rpmconfigdir}/elfdeps --provides %{?__filter_GLIBC_PRIVATE:--filter-private}
 %__elf_requires		%{_rpmconfigdir}/elfdeps --requires %{?__filter_GLIBC_PRIVATE:--filter-private}
-%__elf_magic		^(sticky )?ELF (32|64)-bit.*$
+%__elf_magic		^(setuid )?(setgid )?(sticky )?ELF (32|64)-bit.*$
 %__elf_flags		exeonly
